config/routes.rb

```diff
@@ -5,6 +5,10 @@
 
   get '/auth/facebook', as: :facebook_login
   get '/auth/facebook/callback', to: "sessions#create", as: :facebook_callback
+
+  get '/admin/users', to: 'users#index'
```

Comment:
i would have liked to see this namespaced to "admin" - this will help organize your files and make it easier to read. someone reading your code base will easily know what functionality is scoped to an admin